src/components/views/rooms/EventTile.tsx

```diff
@@ -391,6 +391,8 @@ export default class EventTile extends React.Component<IProps, IState> {
         this.context = context;
         const thread = this.thread;
 
+        this.onReactionsCreated = this.onReactionsCreated.bind(this);
```

Comment:
Why the change to a function here?  Was this an experiment around the event listener we looked at, that is no longer needed?